e2e/flags.go

```diff
@@ -60,13 +60,9 @@ var KubernetesVersion = flag.String("kubernetes-version", "1.21",
 	"The version of Kubernetes to create")
 
 // MultiRepo enables running the tests against multi-repo Config Sync.
-var MultiRepo = flag.Bool("multirepo", false,
+var _ = flag.Bool("multirepo", true,
 	"If true, configure multi-repo Config Sync. Otherwise configure mono-repo.")
```

Comment:
Do we still need the flag?

Reply:
We need it so that `go test` doesn't fail if you pass in the flag. We can remove the flag once there are no CI jobs using the flag

Reply:
ACKk